singer_encodings/csv.py

```diff
@@ -19,10 +19,20 @@ def get_row_iterator(iterable, options=None):
     which can be used to yield CSV rows."""
     options = options or {}
 
+    quoting = None
+    if 'quoting' in options:
+        quoting = eval(options.get('quoting'))
```

Comment:
You typically shouldn't use `eval` since it's just arbitrary code execution and thus a risk vector. Though, in this case, it's only reading from configuration files we're providing.

It might be a bit annoying to use a mapping dict but it is the safest approach.

```python
def get_csv_quoting(options):
  quoting = str(options.get('quoting', "MINIMAL")).upper()
  quoting_choices = {
      "NONE": csv.QUOTE_NONE,
      "MINIMAL": csv.QUOTE_MINIMAL,
      "ALL": csv.QUOTE_ALL,
      "NONNUMERIC": csv.QUOTE_NONNUMERIC
  }
  
  assert quoting in quoting_choices, f"Quoting must be in {quoting_choices.keys()}, given: {quoting}"
  
  return quoting_choices[quoting]
```

Reply:
As an aside, maybe add a test or two? It appears to be using the built-in unittest framework: https://docs.python.org/3/library/unittest.html